lms/djangoapps/instructor/views/registration_codes.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
If there is not enough time to implement a proper API, I at least need to see a `TODO` linking to a tech debt ticket so that we can eventually clean this up.